Extension/src/LanguageServer/referencesProvider.ts

```diff
@@ -0,0 +1,97 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+'use strict';
+import * as vscode from 'vscode';
+import * as util from '../common';
+import { Model, FileItem, ReferenceItem, ReferenceTypeItem } from './referencesModel';
+import { ReferenceInfo, ReferenceType, convertReferenceTypeToString } from './references';
+
+function getReferenceTypeIconPath(referenceType: ReferenceType): vscode.ThemeIcon {
+    // TODO: return icon path for light and dark themes based on reference type
+    switch (referenceType) {

```

Comment:
ConfirmationInProgress is missing. The icon should change depending on if the operation is canceled or not. VS doesn't appear to use any icon for this though, even though it's a type.

Reply:
Will add icon for ConfirmationInProgress later.